### PR TITLE
Add order_by, Get Ready for Deployment

### PR DESCRIPTION
**Pull request by ykostov**
- Add tag/show `websites` order_by `priority`
- add not signed user warning on liking (You must sign in to..)
- fixed warings & bugs, deleted unnecessary alias 
- get ready for deploy -> `gamma version v.1.3.0`
---
- (15.06.2021) WIP - Top 3 websites order by likes -> index page, will continue on another branch in gamma version 
- (02.06.2021) WIP - Favourite Tags on index page -> will continue on another branch in gamma version 
- (21.06.2021) WIP - Setup Deployment config with DigitalOcean

### CODE REVIEW

File: lib/bgsite_official_web/controllers/tag_controller.ex [hunk 1]
```diff
@@ -28,7 +29,7 @@ defmodule BgsiteOfficialWeb.TagController do
   end
 
   def show(conn, %{"id" => id}) do
-    tag = Categories.get_tag!(id)
+    tag = Categories.get_tag_2!(id)
```
Comment (rmilushev) [inline lib/bgsite_official_web/controllers/tag_controller.ex:32]: Why?

File: lib/bgsite_official/categories.ex [hunk 2]
```diff
@@ -41,6 +41,23 @@ defmodule BgsiteOfficial.Categories do
...
   end
 
+def get_tag_2!(id) do
+  query =
+    from(
+      t in Tag,
+      where: t.id == ^id,
+      select: t,
+      preload: [
+        :websites,
```
Comment (rmilushev) [inline lib/bgsite_official/categories.ex:51]: Please, remove this, it is duplicated.

**Review by rmilushev — CHANGES_REQUESTED** [review]

File: lib/bgsite_official_web/controllers/tag_controller.ex (hunk 1 above)
Comment (ykostov) [reply lib/bgsite_official_web/controllers/tag_controller.ex:32]: coz i wanna have 2 definitions of `Repo.get_tag` - a default one:
```
  def get_tag!(id) do
    Repo.get!(Tag, id)
    |> Repo.preload(:websites)
  end
```
and one with `order_by` function:
```
def get_tag_2!(id) do
  query =
    from(
      t in Tag,
      where: t.id == ^id,
      select: t,
      preload: [
        :websites,
        websites:
          ^from(
            w in Websites,
            order_by: [desc: w.priority]
          )
      ]
    )
  Repo.one!(query)
end
```

:)

Comment (ykostov) [reply lib/bgsite_official_web/controllers/tag_controller.ex:32]: However, will change the def name, its inaccurately

**Pull request closed without merging**